shell/platform/tizen/tizen_view_elementary.cc

```diff
@@ -121,12 +124,18 @@ void TizenViewElementary::RegisterEventHandlers() {
   evas_object_callbacks_[EVAS_CALLBACK_RESIZE] =
       [](void* data, Evas* evas, Evas_Object* object, void* event_info) {
         auto* self = reinterpret_cast<TizenViewElementary*>(data);
-        if (self->view_) {
+        if (self->view_delegate_) {
           if (self->container_ == object) {
-            int32_t width = 0, height = 0;
-            evas_object_geometry_get(object, nullptr, nullptr, &width, &height);
+            int32_t x = 0, y = 0, width = 0, height = 0;
+            evas_object_geometry_get(self->container_, &x, &y, &width, &height);
+
+            evas_object_size_hint_min_set(self->container_, width, height);
+            evas_object_size_hint_max_set(self->container_, width, height);
+
+            EvasObjectResizeWithMinMaxHint(self->image_, width, height);
+            evas_object_move(self->image_, x, y);
 
-            self->view_->OnResize(0, 0, width, height);
+            self->view_delegate_->OnResize(x, y, width, height);
```

Comment:
You don't need x and y here.
The geometry of the container is coordinates based on the window.
The coordinates called with OnResize must be 0,0 in the upper left corner of the container.

Reply:
I updated PR. please review again.